kimchi/src/bench.rs

```diff
@@ -34,7 +34,7 @@ pub struct BenchmarkCtx {
 
 impl BenchmarkCtx {
     pub fn srs_size(&self) -> usize {
-        math::ceil_log2(self.index.srs.max_degree())
+        math::ceil_log2(self.index.srs.max_poly_size())
```

Comment:
This is another change. I got rid of the method `max_degree` which was exactly the same than `max_poly_size`.